common/sass/variables/tables.scss

```diff
@@ -1,8 +1,12 @@
 $table-cell-padding: $space-sm;
 $table-condensed-cell-padding: $space-xs;
 $table-bg: transparent;
-$table-bg-accent: #f5f5f5;
-$table-bg-hover: $gray-lightest;
+$table-bg-accent: rgba(0, 0, 0, 0.03);
+$table-bg-hover: rgba(0, 0, 0, 0.06);
 $table-bg-active: $table-bg-hover;
 $table-border-color: transparent;
 $table-cell-padding: 0.75rem 1rem;
+
+tr {
+  transition: background-color 120ms;
+}
```

Comment:
Style in a variables file will cause this to be repeated every time someone `@include`'s it. This should be in an overrides or something.

Also, while a transition is fairly innocuous, we should try to hide opinionated element selectors behind Bootstrap's classes for styled things: http://getbootstrap.com/docs/3.3/css/#tables.